subsys/nrf_security/src/drivers/cracen/cracenpsa/src/common.c

```diff
@@ -570,7 +583,54 @@ static int cracen_prepare_ik_key(const uint8_t *user_data)
 		nrf_cracen_seedram_lock_enable_set(NRF_CRACEN, true);
 	}
 #endif
-	return sx_pk_ik_derive_keys(NULL);
+	struct sx_pk_config_ik cfg;
```

Comment:
This is only applicable to the nRF54H20 right? Does it make not make sense to guard this based on the SOC?

Reply:
done

Reply:
I will approve, if you want  also add the ifdef in the definitions of the domains. Just to make it more clear what their scope is.